src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -575,7 +575,13 @@ const useGroupInfo = (channelId = null) => {
   };
   const handlePressContact = async (item) => {
     const isAnonymousUser = Boolean(item?.user?.anon_user_info_emoji_name);
-    if (item?.user_id === profile?.myProfile?.user_id) return;
+    if (
+      item?.userId === signedProfileId ||
+      item?.userId === anonProfileId ||
+      item?.id === signedProfileId ||
+      item?.id === anonProfileId
+    )
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition checks for both `userId` and `id` fields in the `item` object. If these fields are always expected to be the same, it would be redundant to check both. If they can be different, please ensure that this is the intended behavior. Also, consider adding a comment explaining why both checks are necessary for future reference.

```diff
-    if (
-      item?.userId === signedProfileId ||
-      item?.userId === anonProfileId ||
-      item?.id === signedProfileId ||
-      item?.id === anonProfileId
-    )
+    // Check if the user is the signed or anonymous user
+    const isSignedOrAnonUser = [signedProfileId, anonProfileId].includes(item?.userId || item?.id);
+    if (isSignedOrAnonUser)
```
This change simplifies the condition by using an array and the `includes` method, which makes the code more readable and maintainable.


<!-- This is an auto-generated comment by OSS CodeRabbit -->